testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -2643,3 +2642,500 @@ fn signer_set_rollover() {
         assert!(signer.stop().is_none());
     }
 }
+
+#[test]
+#[ignore]
+// This test involves two miners, each mining tenures with 6 blocks each.
```

Comment:
Addendum: each miner runs signers representing half of the signing power, so what this is really testing is that signers' messages make their way to the active miner.

Reply:
Added in 3e4b875ca8ac7f920810e4ed5de876efd66ac00b.